### DISCUSSION TITLE
Is there any way to replay a game log?

### DISCUSSION THREAD

**Discussion by kumarajith** [boardgameio/boardgame.io#976] [Q&A]
The use case is to be able to view a previously played match and be able to look at every move. I can get the game log from the server. Is there any built in way to do this? 

One way I can think of is to just go through the logs myself and play the moves

**Comment by delucis** [comment]
@kumarajith There’s an open feature request for this #892. It should be possible to build your own version adapting the logic in the debug panel, but even better would be if we could move the log stepping to the client, with an API similar to the one discussed in that issue.
